# TriTrybDB
Pipelines and database information from TritrybDB

## Information access 
https://tritrypdb.org/tritrypdb/app
->Data -> analisis method 
- https://tritrypdb.org/tritrypdb/app/static-content/methods.html

### Methods and Pipelines
- Gene annotation in Ensembl and Ensembl Core: https://www.ensembl.org/info/genome/genebuild/index.html
- TO ANALYSIS  core statistics, DNA feature annotation, external cross reference annotation, RNA gene annotation, repeat feature annotation, and protein feature annotation. CODE https://github.com/AKCLAB/ensembl-production
  * Methods: Methods-Core-Table.pdf
  * Parameters: Methods-Ex-Pipelines.pdf
- In-house genome analyses in Lieu of the EBI Pipeline: Methods-Details-In-Lieu-genomes.pdf
- Proteomics Methods-Proteomics.pdf
- RNA sequences VEuPathDB RNA-Seq alignment.pdf

### Files metadata and studies
- GenomeDataTypes_Summary.csv
- AllDatasets_Summary.csv

## Tools
orfFinder https://github.com/VEuPathDB/ApiCommonData/blob/master/Load/bin/orfFinder
Mercator and MAVID https://github.com/VEuPathDB/ApiCommonData/blob/master/Load/bin/runMercator
DNA repeats https://tandem.bu.edu/trf/home
Signal peptide prediction https://services.healthtech.dtu.dk/services/SignalP-6.0/
Transmembrane domain prediction https://services.healthtech.dtu.dk/services/TMHMM-2.0/
tRNA gene prediction https://lowelab.ucsc.edu/tRNAscan-SE/
OrthoMCL https://github.com/stajichlab/OrthoMCL
